lib/netty

```diff
@@ -1 +1 @@
-Subproject commit 4ae8bdc6ec8d37106f386a7d667a9cd279a8f5d8
+Subproject commit 1cce998bb06f42ff47925390872dcc5c487bbf59
```

Comment:
This looks like a stray change, and will break the build. You likely didn't do a "git submodule update" and then did a "git commit -a". It'll be nice when we don't have to mess with the submodules...